src/app/Fake.Runtime/SdkAssemblyResolver.fs

```diff
@@ -149,14 +149,16 @@ type SdkAssemblyResolver(logLevel: Trace.VerboseLevel) =
                 |> Async.AwaitTask
                 |> Async.RunSynchronously
                 |> List.ofSeq
-                |> List.find (fun product ->
+                |> List.filter (fun product ->
                     this.SdkVersionRaws
                     |> List.exists (fun raws -> product.ProductVersion.Equals raws))
 
-            sdkVersionReleases.GetReleasesAsync()
-            |> Async.AwaitTask
-            |> Async.RunSynchronously
-            |> List.ofSeq
+            sdkVersionReleases
+            |> List.collect (fun rel ->
+                rel.GetReleasesAsync()
+                |> Async.AwaitTask
+                |> Async.RunSynchronously
+                |> List.ofSeq)
```

Comment:
why not?
```suggestion
            |> Seq.map (fun rel -> rel.GetReleasesAsync())
            |> Task.WaitAll
            |> Seq.collect
            |> Seq.toLost
```

Reply:
True that could be better. I expect the list to be very small so no firewall blocks on parallelism. The Seq.toList's side-effected brother Seq.toLost :-D

Reply:
This needs a bit more refactoring, because Tasl.WaitAll is not not returning the result and WhenAll is not waiting the task.